tgui/packages/tgui/interfaces/Radio/index.tsx

```diff
@@ -0,0 +1,135 @@
+                            setFrequency(value, false)}
+                          onChange={(_e: any, value: number) =>
+                            setFrequency(value, true)}
+                        />
+                      )}
+                    </Stack.Item>
+                    <Stack.Item>
+                      <AnimatedNumber
+                        value={data.frequency}
+                        format={formatFrequency}
+                      />
+                    </Stack.Item>
+                  </Stack>
+                </LabeledList.Item>
+              </LabeledList>
+            </Section>
+          </Stack.Item>
+          {data.secureFrequencies.length > 0 && (
+            <Stack.Item grow={1}>
+              <Section title="Supplementary Channels" fill scrollable>
+                <Table>
+                  <Table.Row header>
+                    <Table.Cell header>Channel</Table.Cell>
+                    <Table.Cell header>Frequency</Table.Cell>
+                    <Table.Cell header>Prefix</Table.Cell>
+                  </Table.Row>
+                  {data.secureFrequencies.map(freq => (
+                    <Table.Row key={freq.frequency}>
+                      <Table.Cell>{freq.channel}</Table.Cell>
+                      <Table.Cell>{freq.frequency}</Table.Cell>
+                      <Table.Cell>
+                        <Box as="code">{freq.sayToken}</Box>
+                      </Table.Cell>
+                    </Table.Row>
+                  ))}
+                </Table>
+              </Section>
+            </Stack.Item>
+          )}
+          {!!data.modifiable && (
+            <Stack.Item>
+              <Section title="Access Panel">
+                <LabeledList>
+                  <LabeledList.Item label="Green Wire" labelColor="green">
+                    <Button
+                      color="green"
+                      onClick={() =>
+                        act("toggle-wire", { wire: RadioWires.Transmit })}
+                    >
+                      {data.wires & RadioWires.Transmit ? "Cut" : "Mend"}
+                    </Button>
+                  </LabeledList.Item>
+                  <LabeledList.Item label="Red Wire" labelColor="red">
+                    <Button
+                      color="red"
+                      onClick={() =>
+                        act("toggle-wire", { wire: RadioWires.Receive })}
+                    >
+                      {data.wires & RadioWires.Receive ? "Cut" : "Mend"}
+                    </Button>
+                  </LabeledList.Item>
+                  <LabeledList.Item label="Blue Wire" labelColor="blue">
+                    <Button
+                      color="blue"
+                      onClick={() =>
+                        act("toggle-wire", { wire: RadioWires.Signal })}
+                    >
+                      {data.wires & RadioWires.Signal ? "Cut" : "Mend"}
+                    </Button>
+                  </LabeledList.Item>
```

Comment:
I feel like making the wires part of the data at some point would be good. I understand this is not needed with the current implementation.